workflows/qc/quality-check-standard.wdl

```diff
@@ -69,6 +71,8 @@ workflow quality_check {
         coverage_labels: "An array of equal length to `coverage_beds` which determines the prefix label applied to the output files. If omitted, defaults of `regions1`, `regions2`, etc. will be used."
         gtf: "Gzipped GTF features file. **Required** for RNA-Seq data"
         star_log: "Log file generated by the RNA-Seq aligner STAR. Will be added to MultiQC report."
+        output_intermediate_files: "Output intermediate files? FastQs and if RNA a collated BAM. *WARNING* these files can be large and your backend may copy these files instead of linking to the files that already exist."
```

Comment:
@adthrasher I believe this is true. Pretty sure Cromwell used to copy _everything_. Besides Cromwell, I wouldn't be surprised if other backends do a hardcopy, but I don't know of any for sure. So I could be making up this warning. Do we leave it in?

Reply:
```suggestion
        output_intermediate_files: "Output intermediate files? FastQs and if RNA a collated BAM. *WARNING* these files can be large."
```

I would probably just leave it like this. It's still true and if they use a backend that duplicates the files, they still have a problem.